taggui/utils/settings.py

```diff
@@ -1,6 +1,7 @@
 from PySide6.QtCore import QSettings
 
 # Defaults for settings that are accessed from multiple places.
+# Settings are stored in: Linux: ~/.config/taggui
```

Comment:
This is where the settings are stored: https://doc.qt.io/qtforpython-6/PySide6/QtCore/QSettings.html#locations-where-application-settings-are-stored